files/zh-cn/learn/getting_started_with_the_web/css_basics/index.md

```diff
@@ -118,7 +118,7 @@ p, li, h1 {
 
     > **备注：** CSS 文档中所有位于 `/*` 和 `*/` 之间的内容都是 CSS 注释，它会被浏览器在渲染代码时忽略。你可以在这里写下对你现在要做的事情有帮助的笔记。
 
-    > **备注：** ` /*``*/ ` 不可嵌套，`/*这样的注释是/*不行*/的*/`。CSS 不接受 `//` 注释。
+    > **备注：** `/*` `*/` 不可嵌套，`/*` 这样的注释是 `/*` 不行 `*/` 的 `*/`。CSS 不接受 `//` 注释。
```

Comment:
Think this one might be off, since there is only one Note when I looked upstream

Reply:
this is an out-of-date document, we may update this one later.